frontend_tests/node_tests/compose.js

```diff
@@ -293,7 +293,7 @@ test_ui("send_message", ({override, override_rewire, mock_template}) => {
 
 test_ui("enter_with_preview_open", ({override, override_rewire}) => {
     mock_banners();
-    override(notifications, "clear_compose_notifications", () => {});
+    notifications.clear_compose_notifications = () => {};
```

Comment:
I don't think it's safe to do this sort of monkey-patching in tests without some version of `override` -- I think you're just monkey-patching the module in a way that will affect all future tests with no way to undo. 

Could be wrong -- @andersk can confirm.